src/likes/likes.service.ts

```diff
@@ -0,0 +1,81 @@
+import { HttpException, HttpStatus, Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Like } from 'src/db/entities/like.entity';
+import { Tweet } from 'src/db/entities/tweet.entity';
+import { User } from 'src/db/entities/user.entity';
+import { Repository } from 'typeorm';
+import { AddRemoveLikeDto } from './dto/add-remove-like.dto';
+
+@Injectable()
+export class LikesService {
+  constructor(
+    @InjectRepository(Tweet) private tweetRepository: Repository<Tweet>,
+    @InjectRepository(User) private userRepository: Repository<User>,
+    @InjectRepository(Like) private likeRepository: Repository<Like>,
+  ) {}
+
+  async addRemoveLike(dto: AddRemoveLikeDto): Promise<Like[]> {
+    const foundedUser: User | null = await this.userRepository.findOne({ where: { userId: dto.userId } });
+
+    if (!foundedUser) {
```

Comment:
I guess it would make more sense to call variables like this one just user, tweet, like etc.

If (!user)
if (!tweet)